new-log-viewer/src/services/decoders/ClpIrDecoder.ts

```diff
@@ -31,19 +34,42 @@ class ClpIrDecoder implements Decoder {
         return this.#streamReader.getNumEventsBuffered();
     }
 
-    buildIdx (beginIdx: number, endIdx: number): Nullable<LogEventCount> {
+    // eslint-disable-next-line class-methods-use-this
+    getFilteredLogEventMap (): FilteredLogEventMap {
+        // eslint-disable-next-line no-warning-comments
+        // TODO: Update this after log level filtering is implemented in clp-ffi-js
+        console.error("getFilteredLogEventMap not implemented for IR decoder.");
```

Comment:
```suggestion
        console.error("ClpIrDecoder: getFilteredLogEventMap not implemented.");
```

Ditto for the other log.